src/java/picard/analysis/MeanQualityByCycle.java

```diff
@@ -52,9 +52,23 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Program to generate a data table and pdf chart of " +
-                "mean base quality by cycle from a SAM or BAM file.  Works best on a single lane/run of data, but can be applied to" +
-                "merged BAMs. Uses R to generate chart output.",
+        usage = "This metric gives an overall snapshot of sequencing machine performance, as base quality scores" +
+                " tend to decrease towards the end of a run.  If quality drops precipitously towards the end of the" +
+                " run, the bases of those cycles can be trimmed from the reads. <br /><br />" +
```

Comment:
We don't want to encourage people to trim reads manually. The downstream tools will take quality into account.